doc/holochain_101/src/new_project.md

```diff
@@ -0,0 +1,21 @@
+# Create A New Project
+
+The command line tools discussed in the last article can be used to easily create a new folder on your computer, that contains all the initial folders and files needed for a Holochain application. 
+
+You will typically want to start a new Holochain application this way, since creating them all by hand would waste your time.
+
+In your terminal, change directories to one where you wish to initialize a new Holochain app. The command will create a new folder within the current directory for your app.
+
+Come up with a name for your application, or at least for your project folder.
+
+Copy or type everything after the `$` sign below into your terminal, except replace `your_app_name` with the name you came up with. Press `Enter` to execute the command.
+
+```shell
+$ hcdev init your_app_name
+```
+
+`hcdev` specifies that you wish to use the Holochain command line tools. `init` specifies to use the command for initializing a new project folder. `your_app_name` is an argument you supply as the app, and folder name.
```

Comment:
should we give an example of how to name a project? ie use all lowercase, no spaces

Reply:
thanks Philip, yeah, we should likely add a clarification about what pattern to use for a suitable name